apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/RequestHeaders.java

```diff
@@ -0,0 +1,157 @@
+package gov.cms.bfd.server.war.stu3.providers;
```

Comment:
Will you consider refactoring this to be included in the ../commons so it can be shared across V1 (stu3) and V2 (R4)?